Library/Homebrew/test/test_utils.rb

```diff
@@ -206,4 +206,13 @@ def test_number_readable
     assert_equal "1,000", number_readable(1_000)
     assert_equal "1,000,000", number_readable(1_000_000)
   end
+
+  def test_user_agent
+    ENV.delete "CI"
+    ENV.delete "JENKINS_URL"
+    refute_includes homebrew_user_agent, "CI"
+    ENV["CI"] = "true"
```

Comment:
Really minor nit and doesn't change the outcome of the test, but I believe most CI systems set this to `1` instead.